### PR TITLE
register handlebars helpers

### PR DESCRIPTION
**Pull request by queckezz**
I took a stab at #93. This PR allows a `meta.js` to be defined as a metadata file and additionally allows you to define handlebars helpers like this:

```js
// meta.js
module.exports = {
  helpers: {
    lowercase: str => str.toLowerCase()
  }
}
```

I also tried to document `meta.js` but i'm not sure if this is the best way. Let me know!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ vue init ~/fs/path/to-custom-template my-project
 
 - A template repo **must** have a `template` directory that holds the template files.
 
-- A template repo **may** have a `meta.json` file that provides metadata for the template. The `meta.json` can contain the following fields:
+- A template repo **may** have a metadata file for the template which can be both a `meta.js` or a `meta.json` file. It can contain the following fields:
```
Comment (yyx990803) [inline README.md]: perhaps should use "which can be **either** a ... or ..."

File: README.md [hunk 2]
```diff
@@ -129,6 +129,24 @@ Two commonly used Handlebars helpers, `if_eq` and `unless_eq` are pre-registered
...
+
+Upon registration, they can be used as follows:
+
+``` handlebars
+{{ lowercase name }}
+```
+
 #### File filters
 
 The `filters` field in `meta.json` should be an object hash containing file filtering rules. For each entry, the key is a [minimatch glob pattern](https://github.com/isaacs/minimatch) and the value is a JavaScript expression evaluated in the context of prompt answers data. Example:
```
Comment (yyx990803) [inline README.md]: should also update `meta.json` here to be more generic?

**Comment by yyx990803** [comment]
Nice work! Just a few small doc tweaks - LGTM otherwise.

**Comment by queckezz** [comment]
Alright, fixed :)

**Pull request merged**